playwright_tests/Toaster.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+
+test('toast should disappear after a button click', async ({ page }) => {
+	await page.goto('/Toaster');
+	await page.getByRole('button', { name: 'Delete' }).click();
```

Comment:
You probably want to add a check here that the text "Changes successfully saved." is present.

I'd also create a const for the expected text so that you can test that is is both shown and removed.